extensions/healthie/actions/sendFormCompletionRequest.ts

```diff
@@ -26,6 +26,72 @@ const fields = {
     type: FieldType.STRING,
     required: true,
   },
+  is_recurring: {
+    id: 'is_recurring',
+    label: 'Is recurring',
+    description: 'Set to true if the Form completion should be recurring.',
+    type: FieldType.BOOLEAN,
+    required: false,
+  },
+  frequency: {
+    id: 'frequency',
+    label: 'Frequency',
+    description:
+      'Required if "Is recurring" is set to true. Valid options are: Daily, Weekly, Monthly.',
+    type: FieldType.STRING,
+    required: false,
+  },
+  period: {
+    id: 'period',
+    label: 'Period',
+    description: 'AM or PM.',
+    type: FieldType.STRING,
+    required: false,
+  },
+  hour: {
+    id: 'hour',
+    label: 'Hour',
+    description:
+      'For instance, if you want to trigger the completion request at 1:05 PM, use "1".',
+    type: FieldType.NUMERIC,
+    required: false,
+  },
+  minute: {
+    id: 'minute',
+    label: 'Minute',
+    description:
+      'For instance, if you want to trigger the completion request at 1:05 PM, use "5".',
+    type: FieldType.NUMERIC,
+    required: false,
+  },
+  weekday: {
+    id: 'weekday',
+    label: 'Weekday',
+    description: 'Use the full weekday name, e.g. "Monday".',
+    type: FieldType.STRING,
+    required: false,
+  },
+  monthday: {
+    id: 'monthday',
+    label: 'Monthday',
+    description: 'Number of the day of month, e.g. "27th".',
+    type: FieldType.STRING,
+    required: false,
+  },
+  recurrence_ends: {
+    id: 'recurrence_ends',
+    label: 'Recurrence ends',
+    description: 'Set to true if the recurrence should have an end date.',
+    type: FieldType.BOOLEAN,
+    required: false,
+  },
```

Comment:
Can we omit this and set it to true if the optional recurrence end date below is provided? So two options:
- end date not provided -> `recurrence_ends` is false, we do not pass `ends_on`
- end date provided -> `recurrence_ends` is true, we pass along the value provided

Reply:
Sure, I'll do that